_episodes_rmd/03-day2.Rmd

```diff
@@ -1,150 +1,170 @@
-> 
-> ```{r, eval=TRUE}
-> crs(mintemp)
-> crs(maxtemp)
-> ```
-> 
-> Yes, the CRS for both rasters is the same. Both are in longlat projection, datum WGS84.
-{: .solution}
-
-
-
-## 3) Visualize
-
-Create visualizations of both minimum and maximum temperature rasters using the `ggplot2` package.
-
-
-> ## Solution
-> ```{r prep}
-> mintemp_df <- as.data.frame(mintemp, xy=TRUE)
-> maxtemp_df <- as.data.frame(maxtemp, xy=TRUE)
-> ```
-> 
-> ```{r mintemp}
-> ggplot() +
->    geom_raster(data = mintemp_df , aes(x = x, y = y, fill = mintemp_monthcold_wi)) +
->    scale_fill_viridis_c() +
->    coord_quickmap()
-> ```
->
-> ```{r maxtemp}
-> ggplot() +
->    geom_raster(data = maxtemp_df , aes(x = x, y = y, fill = maxtemp_monthwarm_wi)) +
->    scale_fill_viridis_c() +
->    coord_quickmap()
-> ```
-{: .solution}
-
-### Bonus
-
-You can add the following aesthetics to a `ggplot` to customize the backgrounds, axis labels, title, legend label, and move the legend position to the bottom of the plot:
-
-
-
-```{r, eval=FALSE}
-ggplot() +
-    geom_*() +
-    labs(title = "Your Title", x="X Label", y="Y Label", fill="Legend Label") +
-    theme_bw() +
-    theme(plot.title = element_text(hjust = 0.5), legend.position = "bottom")
-
-```
-
-Explore what these aesthetics do with your plot of maximum monthly temperature.
-
-
-> ## Solution
-> ```{r}
-> ggplot() +
->    geom_raster(data = maxtemp_df , aes(x = x, y = y, fill = maxtemp_monthwarm_wi)) +
->    scale_fill_viridis_c() +
->    coord_quickmap() +
->    labs(title = "Maximum Monthly Temperature: WI", x="Longitude", y="Latitude", fill="Max. Temp (C)") +
->    theme_bw() +
->    theme(plot.title = element_text(hjust = 0.5), legend.position = "bottom")
-> ```
-{: .solution}
-
-
-
-
-## 4) Develop insights
-
-Describe what patterns you observe across the state for coldest and warmest monthly temperatures in Wisconsin. What do you hypothesize is the effect of Lake Michigan?
-
-
-
-> ## Solution
-> Warmer temperatures are in Wisconsin tend toward the lower half of the state and nearer to the Minnesota border. Colder temperatures tend toward the North and near Lake Michigan, possibly due to the lake effect
->
-{: .solution}
-
-## 5) Reproject
-
-Reproject the maxtemp raster into UTM coordinates using the following crs string: ` +proj=utm +zone=15 +datum=WGS84 +units=m +no_defs +ellps=WGS84 +towgs84=0,0,0 `
-
-
-> ## Solution
-> ```{r}
-> maxtemp_utm <- projectRaster(maxtemp, crs=" +proj=utm +zone=15 +datum=WGS84 +units=m +no_defs +ellps=WGS84 +towgs84=0,0,0 ")
-> crs(maxtemp_utm)
-> crs(maxtemp)
-> ```
+---
+title: "Day 2: Introduction to Raster Data"
+exercises: 5
+questions:
+- "What is the coordinate reference system for the maximum temperature raster? Is it the same as the coordinate reference system for the minimum temperature raster? "
+objectives:
+- "Visually explore the patterns in minimum and maximum monthly temperature differs across Wisconsin.  Reproject the maxtemp raster into UTM coordinates."

```

Comment:
We had discussed keeping the questions, objectives, and keypoints generic to any data rather than specific to the capstone data.